packages/medusa/src/api/routes/admin/gift-cards/create-gift-card.ts

```diff
@@ -87,15 +87,14 @@ import { EntityManager } from "typeorm"
  *     $ref: "#/components/responses/500_error"
  */
 export default async (req, res) => {
-  const validated = await validator(AdminPostGiftCardsReq, req.body)
-
+  const validatedBody: AdminPostGiftCardsReq = req.validatedBody
```

Comment:
suggestion: I would add that to the type ` & { balance?: number }` and then 
`validatedBody.balance = validatedBody.value`. this will avoid a copy object that we don't really need here